docs/contributors/css-build-system.md

```diff
@@ -0,0 +1,25 @@
+# CSS build system
```

Comment:
It was confusing for me to find the relevant code that is responsible for this build in webpack, would it make sense to include such files and/or the PR that introduced them?